src/components/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import icon from "../assets/images/night.png";
+import style from "./Header.module.scss";
+import iconMoon from "../assets/images/moon.png";
+import { useContext } from "react";
+import { CustomContext } from "../helpers/Context";
```

Comment:
Let's create a `contexts` folder. It isn't a regular helper.

Reply:
ok